### PR TITLE
update grpc and protobuf versions

### PR DESCRIPTION
**Pull request by vdrobnyi**
Signed-off-by: Victor Drobny <vdrobny@yandex.ru>

### Description of the Change
Protobuf and grpc versions were updated to the latest releases.

### CODE REVIEW

**Comment by l4l** [comment]
There was pretty long discussion about the tags checkout, though I'm up for your way.
Minus of tag: can be changed (-> code might be broken, although it usually doesn't break changes)
Minus of hash: non informative (comments may help)
ping @lebdron @muratovv

File: cmake/Modules/Findgrpc.cmake [hunk 1]
```diff
@@ -26,7 +26,7 @@ find_package_handle_standard_args(grpc DEFAULT_MSG
     )
 
 set(URL https://github.com/grpc/grpc)
-set(VERSION bfcbad3b86c7912968dc8e64f2121c920dad4dfb)
+set(VERSION v1.10.1)
```
Comment (lebdron) [inline cmake/Modules/Findgrpc.cmake]: Hashes guarantee that commit tagged by the version is not changed. Please use hashes for consistency with other dependencies.

File: shared_model/packages/android/android-build.sh [hunk 2]
```diff
@@ -82,7 +82,7 @@ cp -R ./boost_1_66_0/boost "$DEPS_DIR"/include
 
 # protobuf
 git clone https://github.com/google/protobuf
-(cd ./protobuf ; git checkout b5fbb742af122b565925987e65c08957739976a7)
+(cd ./protobuf ; git checkout v1.10.1)
```
Comment (lebdron) [inline shared_model/packages/android/android-build.sh]: v1.10.1 -> v3.5.1, since this is for protobuf.

**Comment by l4l** [comment]
Please also add comments with the version. It might not that easy to determine it by the hash

File: shared_model/packages/javascript/scripts/install-dependencies.sh [hunk 3]
```diff
@@ -20,8 +20,8 @@ git clone https://github.com/boostorg/boost /tmp/boost; \
     (cd /tmp/boost ; sudo /tmp/boost/b2 link=static cxxflags="-std=c++14" -j "$(getconf _NPROCESSORS_ONLN)" install); \
     rm -rf /tmp/boost
 
-# Protobuf (static)
-git clone --depth 1 --branch v3.5.1 https://github.com/google/protobuf
+# Protobuf (static) v3.5.1
+git clone --depth 1 --branch 106ffc04be1abf3ff3399f54ccf149815b287dd9 https://github.com/google/protobuf
```
Comment (lebdron) [inline shared_model/packages/javascript/scripts/install-dependencies.sh]:
```
warning: Could not find remote branch 106ffc04be1abf3ff3399f54ccf149815b287dd9 to clone.
fatal: Remote branch 106ffc04be1abf3ff3399f54ccf149815b287dd9 not found in upstream origin
```
Should probably just clone and checkout as it is done in other scripts.

**Pull request merged**